src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import { getUser, getFolder } from "../api";
+import "./App.css";
+import NavigationBar from "./NavigationBar";
+import { useEffect, useState } from "react";
+import FolderBar from "./FolderBar";
+import CardSection from "./CardSection";
+import Footer from "./Footer";
+
+function App() {
+  const [profile, setProfile] = useState(null);
+  const [folderName, setFolderName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [profileImage, setProfileImage] = useState("");
+  const [cardList, setCardList] = useState(null);
+
+  useEffect(() => {
+    async function getProFile() {
```

Comment:
컴포넌트 내부에 작성하는 함수는 일반적으로 화살표 함수를 사용해주는게 컨벤션인 것 같아요~
```suggestion
    const getProFile = async () => {
```

Reply:
Profile로 표현되어야 할 것 같네요~
```suggestion
    async function getProfile() {
```

Reply:
생각해보니 유저를 불러오는 것이라서 함수명을 fetchUser 정도로 해주면 괜찮을 것 같아요!